### ISSUE TITLE
New version put only pragmas in file

### ISSUE THREAD

**Issue by klipa-dev** [RyuuGan/sol-merger#38]
It seems that with newest version when I run `npx sol-merge ...` it just append all pragmas to merge file and nothing else

**Comment by RyuuGan** [comment]
Hello @nklipa13,

There was a bug about it in version `3.0.0` and it was compiling like this, and it was fixed in version `3.0.1`. Sometimes npx caches the loaded version and not updates it. Try running next command:

```
npx sol-merger@3.0.1 ...
```

Second option is to run it in debug mode:

```
DEBUG="sol-merger*" sol-merger ...
```

You might see the errors and why it is not parsed.

If this also doesn't work for you I need a reproduction script to understand what is happening.

Best regards,
Valerii Aligorskii.

**Comment by klipa-dev** [comment]
Yes, specific version fixed the problem, thanks a lot
